tests/conftest.py

```
@@ -0,0 +1,211 @@
+def event_loop():
+    """Create an instance of the default event loop for the test session."""
+    loop = asyncio.get_event_loop_policy().new_event_loop()
+    yield loop
+    loop.close()
+
+
+@pytest.fixture
+def test_settings():
+    """Test settings with mock values."""
+    return Settings(
+        TELEGRAM_TOKEN="test_token",
```

Ruff (0.13.1)

Possible hardcoded password assigned to argument: "TELEGRAM_TOKEN"

(S106)